libs/vkd3d/vulkan_procs.h

```diff
@@ -213,6 +213,26 @@ VK_DEVICE_EXT_PFN(vkCmdBindTransformFeedbackBuffersEXT)
 VK_DEVICE_EXT_PFN(vkCmdEndQueryIndexedEXT)
 VK_DEVICE_EXT_PFN(vkCmdEndTransformFeedbackEXT)
 
+#ifdef VKD3D_BUILD_STANDALONE_D3D12
```

Comment:
Again, why the ifdef? Just always load them, and hide the win32-specific functions behind an *appropriate* ifdef (`#ifdef VK_KHR_win32_surface`).

Reply:
Resolved locally.